readme.txt

```diff
@@ -65,11 +65,11 @@ Share options include:
 
 == Changelog ==
 
-= <VERSION> - <DATE> =
+= = 1.1.0 - <DATE> =
```

Comment:
May want to replace <DATE> here to with release date.

Reply:
Yup thanks, the date is not finalized yet.